tests/test_py_counter.py

```diff
@@ -17,12 +17,20 @@ def test_count_comments():
     expected_comments = 14
     assert test_pycount.count_comments() == expected_comments
 
+def test_count_while():
+    """Test case for counting the while loops."""
+    expected_while = 0
+    assert test_pycount.count_while() == expected_while

```

Comment:
Your test input should contain an example of a `while` loop so that you can test its detection.

Reply:
@RyanHiltyAllegheny